src/client/editor/Editor.js

```diff
@@ -1224,7 +1224,11 @@ export default class Editor {
 
   async addGLTFModelNode(name, uri, originUri) {
     const attribution = await this.project.getImportAttribution(uri);
-    this.addUnicomponentNode(name, "gltf-model", { src: uri, attribution, origin: originUri });
+    this.addUnicomponentNode(name, "gltf-model", {
+      src: uri,
+      attribution: JSON.stringify(attribution),
```

Comment:
Does this work for the legacy attribution? Doesn't `getImportAttribution` return an object?

Reply:
Oh I see you parse it later. Interesting.